### PR TITLE
Adds Nicotine Patch Packets, Adds them to Vending Machines

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

This PR adds a unique icon for 9u nicotine patches, a pack for them to go in, and adds them to vending machines! More diversity n shit. 

## Why It's Good For The Game

Its never too late to quit! Adds a different way for smokers to get their fix, however it will not be spawned on their person. Be careful not to take two at once, or smoke, since you could OD.

image

## Changelog
:cl:
imageadd: an icon for nicotine patches, and nicotine patch packs
add: nicotine patches
add: nicotine patch packs are now available in vending machines, don't take more than one at once!
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
I'd like to take this opportunity to call everyone's attention to the fact that this is indeed a webeditless PR.

**Comment by Time-Green** [comment]
im gonna get my enemies addicted to nicotine now

**Comment by carlarctg** [comment]
finally i can take the smoker trait without getting huge mood maluses every time someone breaches the main hallway

File: code/modules/reagents/reagent_containers/patch.dm [hunk 1]
```diff
@@ -44,3 +44,9 @@
 	desc = "Helps with brute and burn injuries."
 	list_reagents = list(/datum/reagent/medicine/synthflesh = 20)
 	icon_state = "bandaid_both"
+
+/obj/item/reagent_containers/pill/patch/nicotine
```
Comment (PKPenguin321) [inline code/modules/reagents/reagent_containers/patch.dm]: fucks me up that patches are still pathed under /pill, thought for a minute that you would just pop the nicotine patches in your mouth 😆

Comment (wesoda25) [reply code/modules/reagents/reagent_containers/patch.dm]: ya i thought it was wack too, they can actually fit in pill bottles bc of it. i saw a piece of code that I think prevented people from being able to eat them so thats whatever

**Comment by nemvar** [comment]
If you want nicotine patches, make them yourself gamer.

**Comment by wesoda25** [comment]
Holy shit you guys really will call anything powergaming. Patches provide a very minimal benefit, if at all. Those 9 units will have been processed in your system way before you’ll have finished a cigarette. Meaning you’ll be out of patches much quicker. So powergaming wise, there’s hardly any benefit. 

But beyond that, this allows races/jobs such as plasmamen and mimes to take the smoker quirk, as they cannot free up their mask slots. It adds more depth to the game, and allows one to pursue different options to get their nic fix. Making patches is boring and lame, this actually puts them into the economy system. 

Also im immune to “LOL POWERGAMER” argument since I only take smoker quirk and no positives with it. For other players above argument applies.

**Comment by nemvar** [comment]
Is it really too much to ask that you make your own nicotine patches? This isn't about powergaming, it's about limiting roundstart options to allow for player expression during the round.

**Comment by wesoda25** [comment]
> Is it really too much to ask that you make your own nicotine patches? This isn't about powergaming, it's about limiting roundstart options to allow for player expression during the round.

How tf does putting patches in a vending machine limit a way for a player to express themselves? 

> Doing stuff isn't boring and lame, not going through the effort to get something is what's lame.

one sec lemme remove cigarettes and force people to grow, dry, and roll up their own tobacco. after all, our system is lame!

**Comment by nemvar** [comment]
Mate, my point is that if you want special snowflake gear, you should make it yourself. The way we create gameplay is by being a sandbox.

My argument is that the sandbox should start as empty as possible while still containing all the ingredients to make a nice sand castle. Where is the fun in having your castle prebuilt?
Sure, you could agaze yourself at it for a while but such enjoyment is superficial at best. What really drives the fun behind creating something is the process and not the result.

In your second point you are basically arguing that because it is possible to add new sand to our box that in turn we should get get our castles for free.
This isn't about patches in vendors, it is about the principal behind adding them.

We have engaging systems for creating custom patches, why shouldn't we force people to use them? What gameplay is this creating? What interaction is it encouraging?
This is basically as if we were to add food that only the chef can currently cook to the vendors. There is no reason behind it, it doesn't encourage interaction, it doesn't encourage more engaging gameplay.

It's not like you can even make a realism/immersion argument here, real cigarette vendors don't usually sell nicotine patches. So your addition has to serve either gameplay or artistic purpose neither of which are justified.

**Comment by wesoda25** [comment]
I get your point but I believe you are using that argument in the wrong situation. If this were a PR that adds every ingredient a chef ever needs to kitchen, I'd get it. Not in this scenario. I don't think of it as snowflake gear since it makes perfect sense to be in vendors. This doesn't degrade from our system of patch making in any way. Its not as if the journey to the holodeck to grind your cigs is one important to gameplay, or encourages individualism.

**Comment by Mickyan** [comment]
It's hard to see this as anything more than a powergaming tool, using cigarettes/making nicotine patches has always been cumbersome enough to not warrant nerfing nicotine, now you're adding readily available anti-stun patches

**Comment by wesoda25** [comment]
Yeah go fuck yourself. "considering the way you play" LOL. you dont even know how I play because I dont interact with you and your shitty fucking moth metagroup. I literally have nothing to gain from this feature, because, as I said, I take smoker with no positive quirks (oooh yeahh im such a fuckign powergamer omg swanni imms ooo badddd). excuse me as i go apply my nicotine patches so i can get half a second less stun in combAT!!! WHICH I COULD ALSO GET MY SMOKING A CIGARETTE. 

literally adding it for diversity and more IC means to pursue ones nicotine addiction, more options are fun. theres hardly any benefit to taking patches as opposed to cigs if at all, im sorry that you are so repulsed at people unable to free up their mask slot being able to take a quirk. At this point I'm only taking nemvars opinion seriously because hes the only one who has managed to justify his point of view other than "LOL powergame". 

and if you think nic is so OP (its not) ill nerf the fuck out of it just for you :)

**Comment by wesoda25** [comment]
enjoy the entertainment until i cooldown and decide to delete that first part

**Comment by Qustinnus** [comment]
fam if ur doing it for rp purposes just go to chemistry and ask for some bro there's your rp problem solved

**Comment by optimumtact** [comment]
just dont' get addicted to nicotine lol

**Pull request closed without merging**